package.json

```diff
@@ -11,10 +11,11 @@
     "node": ">=6"
   },
   "scripts": {
-    "install-all": "npm run install-cli && npm run install-extension && npm run install-viewer",
+    "install-all": "npm run install-cli && npm run install-extension && npm run install-viewer && npm run install-recipes",
     "install-cli": "cd ./lighthouse-cli && npm install",
     "install-extension": "cd ./lighthouse-extension && npm install",
     "install-viewer": "cd ./lighthouse-viewer && npm install",
+    "install-recipes": "cd ./docs/recipes && npm install",
```

Comment:
IMO, we don't need to necessarily make recipes runnable from the main package. I see them as more as documentation examples that users can copy/paste or run themselves. Installing these extras will also slow down our good buddy Travis.